components/ILIAS/Authentication/classes/PageLanguages/PageLanguagesOverviewTable.php

```diff
@@ -18,23 +18,39 @@
 
 declare(strict_types=1);
 
-namespace ILIAS\Authentication\LogoutPage;
+namespace ILIAS\Authentication\PageLanguages;
 
 use ILIAS\Data;
 use ILIAS\UI;
 use ilArrayUtil;
 use Psr\Http\Message\ServerRequestInterface;
 use ilLanguage;
 use ilCtrlInterface;
-use ilAuthLogoutPageEditorSettings;
+use AuthPageEditorContext;
+use ilAuthPageEditorSettings;
 
-class LogoutPageLanguagesOverviewTable implements UI\Component\Table\DataRetrieval
+class PageLanguagesOverviewTable implements UI\Component\Table\DataRetrieval
 {
+    private static array $context_lng_vars =
```

Comment:
We should add a PhpDoc describing the array shape:

`/** @var array<value-of<AuthPageEditorContext>, array{'pages' string, 'page': string, 'param_namspace' string, 'table_action': string}}>*/`